File: VolumeOfCylinder/VolumeOfCylinder.java
import java.util.Scanner;

class VolumeOfCylinder {

    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);

        System.out.println("Enter the radius of the cylinder:");
        double r = s.nextDouble();
        System.out.println("Enter the height of the cylinder::");
        double h = s.nextDouble();

        double volume = getVolumeOfCylinder(r, h);

        System.out.println("The volume of Cylinder is: " + volume);

    }

    private static double getVolumeOfCylinder(double r, double h) {
        return (22 * r * r * h) / 7;
    }
}

File: VolumeOfCylinder/VolumeOfCylinder.c
#include <stdio.h>
#include <math.h>

float calcVolume(float radius, float height){
	return M_PI*radius*radius*height;
}

int main()
{
	float radius, height;

	printf("Enter the radius of cylinder: ");
	scanf("%f", &radius);
	printf("Enter the height of cylinder: ");
	scanf("%f", &height);
	printf("The volume of given cylinder is %f\n", calcVolume(radius, height));
	return 0;
}